integration-tests/README.md

```diff
@@ -54,9 +54,9 @@ Optionally, additional useful options are:
 
 ```sh
 HEADLESS=false \
-ENVIRONMENT=[local|dev|alpha|perf|staging] \
+ENVIRONMENT=[dev|alpha|perf|staging] \
 ```
-By default, the tests will run headless against Terra UI running on your local machine.
+By default, the tests will run against dev.
```

Comment:
Could you add information here about running `test-local` or setting the environment variable? I never used `test-local`, so I know I would have gotten tripped up on this.

Reply:
I might move this documentation to the wiki so that it's easier to update.

Reply:
I removed this README in d817d86504e4c3be53359bcf22f5bba447bbcb72 and moved its content to the wiki (https://github.com/DataBiosphere/terra-ui/wiki/Integration-tests).